src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/MethodQueryIT.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.esintgtest;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Locale;
+
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.containsString;
+
+/**
+ * 定製方法查詢．
+ * @author ansj
+ *
+ */
+public class MethodQueryIT extends SQLIntegTestCase {
```

Comment:
seems the indentation is not consistent in this file, please fix.

Reply:
also, can you break the lines longer than 120 characters? Maybe we have 140 in current config, but I think 120 is reasonable. This is to make sure that we don't introduce more and more checkstyle warnings/errors.

Reply:
Done.